bundle_resources/Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>English</string>
+	<key>CFBundleExecutable</key>
+	<string>ChiraEditor</string>
```

Comment:
If this folder is specific to the editor, can it be moved and renamed to `resources/editor/macos`?

The folder `resources/editor/icon` can be renamed to `windows` or similar

Reply:
yes but that would complicate the resource copying command substantially. having to search for specific files and folders to move, ignore, etc.

Reply:
ig its fine for now, if there was ever another application it would get ugly though